ext/src/dll/CMakeLists.txt

```diff
@@ -22,6 +22,22 @@ if(WITH_OTLP_HTTP)
                         PRIVATE opentelemetry_exporter_otlp_http)
 endif()
 
+target_link_libraries(
+  opentelemetry_cpp PRIVATE opentelemetry_metrics
+                            opentelemetry_exporter_ostream_metrics)
+
+if(WITH_OTLP_GRPC)
+  add_compile_definitions(WITH_OTLP_GRPC)
```

Comment:
Is this duplicated with line 14 above?

Reply:
Yes, an oversight from my end. Fixed in the latest commit